### ISSUE TITLE
Opera Coast discovery

### ISSUE THREAD

**Issue by szuliq** [tobie/ua-parser#478]
It seems like `ua-parser` doesn't recognize [Opera Coast](http://operacoast.com/) browser at all.

An example `User-Agent` header

```
Mozilla/5.0 (iPhone; CPU iPhone OS 7_0_3 like Mac OS X) AppleWebKit/537.51.1 (KHTML, like Gecko) Coast/3.1.0.79792 Mobile/11B511 Safari/7534.48.3
```

gets recognized as:

``` python
{'device': {'family': u'iPhone'},
 'os': {'family': u'iOS',
  'major': u'7',
  'minor': u'0',
  'patch': u'3',
  'patch_minor': None},
 'string': u"'Mozilla/5.0 (iPhone; CPU iPhone OS 7_0_3 like Mac OS X) AppleWebKit/537.51.1 (KHTML, like Gecko) Coast/3.1.0.79792 Mobile/11B511 Safari/7534.48.3'",
 'user_agent': {'family': u'Mobile Safari',
  'major': u'7',
  'minor': u'3',
  'patch': None}}
```

**Comment by patrickkettner** [comment]
@r4fek belive this can be closed as a result of #490 - no?

**Comment by r4fek** [comment]
@patrickkettner yes, of course.
